spring-cloud-kubernetes-fabric8-config/src/test/java/org/springframework/cloud/kubernetes/fabric8/config/Fabric8BootstrapConfigurationTests.java

```diff
@@ -129,4 +130,39 @@ public void secretsOnlyPresent() {
 
 	}
 
+	// tests that @ConditionalOnCloudPlatform(CloudPlatform.KUBERNETES) has the desired
```

Comment:
these two below prove that `@ConditionalOnCloudPlatform(CloudPlatform.KUBERNETES` enable/disable brings or not, the needed beans